Google.PowerShell.IntegrationTests/BigQuery/BigQuery.GcbqDataset.Tests.ps1

```diff
@@ -27,6 +27,17 @@ Describe "Get-GcbqDataset" {
         }
     }
 
+    It "should list datasets and then get complete dataset objects from the DatasetData" {
+        try {
+            New-GcbqDataset "test_id_2"
+            $a = Get-GcbqDataset | Get-GcbqDataset
```

Comment:
This feels silly. If we think the user may want to make this conversion, I would argue for making the extra requests to get the full Dataset object in the first place, and never letting our users see a DatasetList.DatasetsData. @quoctruong What is your opinion about this?